Unable to build on Win 11 (BUILD.md outdated?)

I tried building Brogue on Windows 11 by cloning the master branch and following the instructions in BUILD.md.

After making sure I have gcc in MSYS2 (v13.2.0 was installed as per the MSYS2 website's own guide), I went to the root directory of BrogueCE (where BUILD.md is located) and executed `pacman -S make mingw-w64-x86_64-{gcc,SDL2,SDL2_image}`, which was completed without issues. However, the next step of executing `make bin/brogue.exe` has failed with these errors:

```
make: sdl2-config: No such file or directory
make: sdl2-config: No such file or directory
make: *** No rule to make target 'bin/brogue.exe'.  Stop.
```

It was suggested to me to start the MSYS shell with `msys2_shell.cmd -mingw64`, which took care of the complaint regarding the missing "sdl2-config", but I still saw the `make: *** No rule to make target 'bin/brogue.exe'.  Stop.` error.

Then, I had another tip to simply try `make`, because the makefile defaults to building an exe. With this I was able to build, however there were errors, but it has at least created an executable:

```
Sandor@Legion MINGW64 /c/Users/Sandor/Brogue/github/BrogueCE
$ make
mkdir -p vars
/bin/sh: line 1: cmp: command not found
/bin/sh: line 1: cmp: command not found
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/Architect.c -o src/brogue/Architect.o
src/brogue/Architect.c:968:29: warning: argument 6 of type 'item *[200]' with mismatched bound [-Warray-parameter=]
  968 |                       item *parentSpawnedItems[MACHINES_BUFFER_LENGTH],
      |                       ~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from src/brogue/Architect.c:24:
src/brogue/Rogue.h:2875:33: note: previously declared as 'item *[50]'
 2875 |                           item *parentSpawnedItems[50],
      |                           ~~~~~~^~~~~~~~~~~~~~~~~~~~~~
src/brogue/Architect.c:969:33: warning: argument 7 of type 'creature *[200]' with mismatched bound
-Warray-parameter=]
  969 |                       creature *parentSpawnedMonsters[MACHINES_BUFFER_LENGTH]) {
      |                       ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/brogue/Rogue.h:2876:37: note: previously declared as 'creature *[50]'
 2876 |                           creature *parentSpawnedMonsters[50]);
      |                           ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/Buttons.c -o src/brogue/Buttons.o
src/brogue/Buttons.c: In function 'setButtonText':
src/brogue/Buttons.c:156:5: warning: 'strncpy' output may be truncated copying 299 bytes from a string of length 299 [-Wstringop-truncation]
  156 |     strncpy(button->text, textBuf, BUTTON_TEXT_SIZE - 1);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/Combat.c -o src/brogue/Combat.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/Dijkstra.c -o src/brogue/Dijkstra.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/Globals.c -o src/brogue/Globals.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/GlobalsBase.c -o src/brogue/GlobalsBase.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/Grid.c -o src/brogue/Grid.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/IO.c -o src/brogue/IO.o
src/brogue/IO.c: In function 'printMonsterInfo':
src/brogue/IO.c:4582:13: warning: 'printProgressBar' reading 100 bytes from a region of size 10 [-Wstringop-overread]
 4582 |             printProgressBar(0, y++, "Nutrition", player.status[STATUS_NUTRITION], STOMACH_SIZE, &blueBar, dim);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/brogue/IO.c:4582:13: note: referencing argument 3 of type 'const char[100]'
src/brogue/IO.c:4342:6: note: in a call to function 'printProgressBar'
 4342 | void printProgressBar(short x, short y, const char barLabel[COLS], long amtFilled, long amtMax, const color *fillColor, boolean dim) {
      |      ^~~~~~~~~~~~~~~~
src/brogue/IO.c:4584:13: warning: 'printProgressBar' reading 100 bytes from a region of size 19 [-Wstringop-overread]
 4584 |             printProgressBar(0, y++, "Nutrition (Hungry)", player.status[STATUS_NUTRITION], STOMACH_SIZE, &blueBar, dim);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/brogue/IO.c:4584:13: note: referencing argument 3 of type 'const char[100]'
src/brogue/IO.c:4342:6: note: in a call to function 'printProgressBar'
 4342 | void printProgressBar(short x, short y, const char barLabel[COLS], long amtFilled, long amtMax, const color *fillColor, boolean dim) {
      |      ^~~~~~~~~~~~~~~~
src/brogue/IO.c:4586:13: warning: 'printProgressBar' reading 100 bytes from a region of size 17 [-Wstringop-overread]
 4586 |             printProgressBar(0, y++, "Nutrition (Weak)", player.status[STATUS_NUTRITION], STOMACH_SIZE, &blueBar, dim);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/brogue/IO.c:4586:13: note: referencing argument 3 of type 'const char[100]'
src/brogue/IO.c:4342:6: note: in a call to function 'printProgressBar'
 4342 | void printProgressBar(short x, short y, const char barLabel[COLS], long amtFilled, long amtMax, const color *fillColor, boolean dim) {
      |      ^~~~~~~~~~~~~~~~
src/brogue/IO.c:4588:13: warning: 'printProgressBar' reading 100 bytes from a region of size 18 [-Wstringop-overread]
 4588 |             printProgressBar(0, y++, "Nutrition (Faint)", player.status[STATUS_NUTRITION], STOMACH_SIZE, &blueBar, dim);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/brogue/IO.c:4588:13: note: referencing argument 3 of type 'const char[100]'
src/brogue/IO.c:4342:6: note: in a call to function 'printProgressBar'
 4342 | void printProgressBar(short x, short y, const char barLabel[COLS], long amtFilled, long amtMax, const color *fillColor, boolean dim) {
      |      ^~~~~~~~~~~~~~~~
src/brogue/IO.c:4601:17: warning: 'printProgressBar' reading 100 bytes from a region of size 11 [-Wstringop-overread]
 4601 |                 printProgressBar(0, y++, (monst == &player ? "Levitating" : "Flying"), monst->status[i], monst->maxStatus[i], &redBar, dim);
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/brogue/IO.c:4601:17: note: referencing argument 3 of type 'const char[100]'
src/brogue/IO.c:4342:6: note: in a call to function 'printProgressBar'
 4342 | void printProgressBar(short x, short y, const char barLabel[COLS], long amtFilled, long amtMax, const color *fillColor, boolean dim) {
      |      ^~~~~~~~~~~~~~~~
src/brogue/IO.c:4624:13: warning: 'printProgressBar' reading 100 bytes from a region of size 40 [-Wstringop-overread]
 4624 |             printProgressBar(0, y++,  monsterText[monst->info.monsterID].absorbStatus, monst->corpseAbsorptionCounter, 20, &redBar, dim);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/brogue/IO.c:4624:13: note: referencing argument 3 of type 'const char[100]'
src/brogue/IO.c:4342:6: note: in a call to function 'printProgressBar'
 4342 | void printProgressBar(short x, short y, const char barLabel[COLS], long amtFilled, long amtMax, const color *fillColor, boolean dim) {
      |      ^~~~~~~~~~~~~~~~
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/Items.c -o src/brogue/Items.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/Light.c -o src/brogue/Light.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/MainMenu.c -o src/brogue/MainMenu.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/Math.c -o src/brogue/Math.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/Monsters.c -o src/brogue/Monsters.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/Movement.c -o src/brogue/Movement.o
src/brogue/Movement.c:132:29: warning: argument 1 of type 'char *' declared as a pointer [-Warray-parameter=]
  132 | void describeLocation(char *buf, short x, short y) {
      |                       ~~~~~~^~~
In file included from src/brogue/Movement.c:24:
src/brogue/Rogue.h:3010:32: note: previously declared as an array 'char[79]'
 3010 |     void describeLocation(char buf[DCOLS], short x, short y);
      |                           ~~~~~^~~~~~~~~~
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/PowerTables.c -o src/brogue/PowerTables.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/Recordings.c -o src/brogue/Recordings.o
src/brogue/Recordings.c: In function 'seek':
src/brogue/Recordings.c:775:13: warning: 'printProgressBar' reading 100 bytes from a region of size 21 [-Wstringop-overread]
  775 |             printProgressBar((COLS - 20) / 2, ROWS / 2, "[     Loading...   ]",
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  776 |                              rogue.playerTurnNumber - startTurnNumber,
      |                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  777 |                              targetTurnNumber - startTurnNumber, &darkPurple, false);
      |                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/brogue/Recordings.c:775:13: note: referencing argument 3 of type 'const char[100]'
In file included from src/brogue/Recordings.c:26:
src/brogue/Rogue.h:2920:10: note: in a call to function 'printProgressBar'
 2920 |     void printProgressBar(short x, short y, const char barLabel[COLS], long amtFilled, long amtMax, const color *fillColor, boolean dim);
      |          ^~~~~~~~~~~~~~~~
src/brogue/Recordings.c: In function 'loadSavedGame':
src/brogue/Recordings.c:1374:17: warning: 'printProgressBar' reading 100 bytes from a region of size 21 [-Wstringop-overread]
 1374 |                 printProgressBar((COLS - 20) / 2, ROWS / 2, "[     Loading...   ]", recordingLocation, lengthOfPlaybackFile, &darkPurple, false);
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/brogue/Recordings.c:1374:17: note: referencing argument 3 of type 'const char[100]'
src/brogue/Rogue.h:2920:10: note: in a call to function 'printProgressBar'
 2920 |     void printProgressBar(short x, short y, const char barLabel[COLS], long amtFilled, long amtMax, const color *fillColor, boolean dim);
      |          ^~~~~~~~~~~~~~~~
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/RogueMain.c -o src/brogue/RogueMain.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/SeedCatalog.c -o src/brogue/SeedCatalog.o
src/brogue/SeedCatalog.c: In function 'printSeedCatalogAltars':
src/brogue/SeedCatalog.c:227:21: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 6 [-Wstringop-overflow=]
  227 |                     printSeedCatalogCsvLine(rogue.seed, rogue.depthLevel, 1, "altar", "resurrection altar",
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  228 |                             NO_ENCHANTMENT_STRING, NO_RUNIC_STRING, vaultNumber, NO_OPENS_VAULT_STRING,
      |                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  229 |                             NO_CARRIED_BY_MONSTER_STRING, NO_ALLY_STATUS_STRING, NO_MUTATION_STRING);
      |                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/brogue/SeedCatalog.c:227:21: note: referencing argument 4 of type 'char[50]'
src/brogue/SeedCatalog.c:227:21: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 19 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:227:21: note: referencing argument 5 of type 'char[50]'
src/brogue/SeedCatalog.c:227:21: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:227:21: note: referencing argument 6 of type 'char[50]'
src/brogue/SeedCatalog.c:227:21: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:227:21: note: referencing argument 7 of type 'char[50]'
src/brogue/SeedCatalog.c:227:21: note: referencing argument 8 of type 'char[10]'
src/brogue/SeedCatalog.c:227:21: warning: 'printSeedCatalogCsvLine' accessing 10 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:227:21: note: referencing argument 9 of type 'char[10]'
src/brogue/SeedCatalog.c:227:21: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:227:21: note: referencing argument 10 of type 'char[50]'
src/brogue/SeedCatalog.c:227:21: warning: 'printSeedCatalogCsvLine' accessing 20 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:227:21: note: referencing argument 11 of type 'char[20]'
src/brogue/SeedCatalog.c:227:21: warning: 'printSeedCatalogCsvLine' accessing 100 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:227:21: note: referencing argument 12 of type 'char[100]'
src/brogue/SeedCatalog.c:36:13: note: in a call to function 'printSeedCatalogCsvLine'
   36 | static void printSeedCatalogCsvLine(uint64_t seed, short depth, short quantity, char categoryName[50], char kindName[50],
      |             ^~~~~~~~~~~~~~~~~~~~~~~
src/brogue/SeedCatalog.c:244:17: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 6 [-Wstringop-overflow=]
  244 |                 printSeedCatalogCsvLine(rogue.seed, rogue.depthLevel, 1, "altar", "commutation altar",
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  245 |                                         NO_ENCHANTMENT_STRING, NO_RUNIC_STRING, vaultNumber, NO_OPENS_VAULT_STRING,
      |                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  246 |                                         NO_CARRIED_BY_MONSTER_STRING, NO_ALLY_STATUS_STRING, NO_MUTATION_STRING);
      |                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/brogue/SeedCatalog.c:244:17: note: referencing argument 4 of type 'char[50]'
src/brogue/SeedCatalog.c:244:17: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 18 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:244:17: note: referencing argument 5 of type 'char[50]'
src/brogue/SeedCatalog.c:244:17: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:244:17: note: referencing argument 6 of type 'char[50]'
src/brogue/SeedCatalog.c:244:17: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:244:17: note: referencing argument 7 of type 'char[50]'
src/brogue/SeedCatalog.c:244:17: note: referencing argument 8 of type 'char[10]'
src/brogue/SeedCatalog.c:244:17: warning: 'printSeedCatalogCsvLine' accessing 10 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:244:17: note: referencing argument 9 of type 'char[10]'
src/brogue/SeedCatalog.c:244:17: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:244:17: note: referencing argument 10 of type 'char[50]'
src/brogue/SeedCatalog.c:244:17: warning: 'printSeedCatalogCsvLine' accessing 20 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:244:17: note: referencing argument 11 of type 'char[20]'
src/brogue/SeedCatalog.c:244:17: warning: 'printSeedCatalogCsvLine' accessing 100 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:244:17: note: referencing argument 12 of type 'char[100]'
src/brogue/SeedCatalog.c:36:13: note: in a call to function 'printSeedCatalogCsvLine'
   36 | static void printSeedCatalogCsvLine(uint64_t seed, short depth, short quantity, char categoryName[50], char kindName[50],
      |             ^~~~~~~~~~~~~~~~~~~~~~~
src/brogue/SeedCatalog.c: In function 'printSeedCatalogItem':
src/brogue/SeedCatalog.c:102:9: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 20 [-Wstringop-overflow=]
  102 |         printSeedCatalogCsvLine(rogue.seed, rogue.depthLevel, theItem->quantity, categoryName, kindName, enchantment,
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  103 |                                 runicName, vaultNumber, opensVaultNumber, carriedByMonsterName, NO_ALLY_STATUS_STRING,
      |                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  104 |                                 mutationName);
      |                                 ~~~~~~~~~~~~~
src/brogue/SeedCatalog.c:102:9: note: referencing argument 4 of type 'char[50]'
src/brogue/SeedCatalog.c:102:9: note: referencing argument 5 of type 'char[50]'
src/brogue/SeedCatalog.c:102:9: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 5 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:102:9: note: referencing argument 6 of type 'char[50]'
src/brogue/SeedCatalog.c:102:9: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 30 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:102:9: note: referencing argument 7 of type 'char[50]'
src/brogue/SeedCatalog.c:102:9: note: referencing argument 8 of type 'char[10]'
src/brogue/SeedCatalog.c:102:9: note: referencing argument 9 of type 'char[10]'
src/brogue/SeedCatalog.c:102:9: note: referencing argument 10 of type 'char[50]'
src/brogue/SeedCatalog.c:102:9: warning: 'printSeedCatalogCsvLine' accessing 20 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:102:9: note: referencing argument 11 of type 'char[20]'
src/brogue/SeedCatalog.c:102:9: note: referencing argument 12 of type 'char[100]'
src/brogue/SeedCatalog.c:36:13: note: in a call to function 'printSeedCatalogCsvLine'
   36 | static void printSeedCatalogCsvLine(uint64_t seed, short depth, short quantity, char categoryName[50], char kindName[50],
      |             ^~~~~~~~~~~~~~~~~~~~~~~
src/brogue/SeedCatalog.c: In function 'printSeedCatalogFloorGold':
src/brogue/SeedCatalog.c:185:9: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 5 [-Wstringop-overflow=]
  185 |         printSeedCatalogCsvLine(rogue.seed, rogue.depthLevel, gold, "gold", kindName, NO_ENCHANTMENT_STRING,
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  186 |                                 NO_RUNIC_STRING, NO_VAULT_STRING, NO_OPENS_VAULT_STRING, NO_CARRIED_BY_MONSTER_STRING,
      |                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  187 |                                 NO_ALLY_STATUS_STRING, NO_MUTATION_STRING);
      |                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/brogue/SeedCatalog.c:185:9: note: referencing argument 4 of type 'char[50]'
src/brogue/SeedCatalog.c:185:9: note: referencing argument 5 of type 'char[50]'
src/brogue/SeedCatalog.c:185:9: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:185:9: note: referencing argument 6 of type 'char[50]'
src/brogue/SeedCatalog.c:185:9: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:185:9: note: referencing argument 7 of type 'char[50]'
src/brogue/SeedCatalog.c:185:9: warning: 'printSeedCatalogCsvLine' accessing 10 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:185:9: note: referencing argument 8 of type 'char[10]'
src/brogue/SeedCatalog.c:185:9: warning: 'printSeedCatalogCsvLine' accessing 10 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:185:9: note: referencing argument 9 of type 'char[10]'
src/brogue/SeedCatalog.c:185:9: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:185:9: note: referencing argument 10 of type 'char[50]'
src/brogue/SeedCatalog.c:185:9: warning: 'printSeedCatalogCsvLine' accessing 20 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:185:9: note: referencing argument 11 of type 'char[20]'
src/brogue/SeedCatalog.c:185:9: warning: 'printSeedCatalogCsvLine' accessing 100 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:185:9: note: referencing argument 12 of type 'char[100]'
src/brogue/SeedCatalog.c:36:13: note: in a call to function 'printSeedCatalogCsvLine'
   36 | static void printSeedCatalogCsvLine(uint64_t seed, short depth, short quantity, char categoryName[50], char kindName[50],
      |             ^~~~~~~~~~~~~~~~~~~~~~~
src/brogue/SeedCatalog.c: In function 'printSeedCatalogMonster':
src/brogue/SeedCatalog.c:135:9: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 10 [-Wstringop-overflow=]
  135 |         printSeedCatalogCsvLine(rogue.seed, rogue.depthLevel, 1, categoryName, theMonster->info.monsterName,
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  136 |                                 NO_ENCHANTMENT_STRING, NO_RUNIC_STRING, NO_VAULT_STRING, NO_OPENS_VAULT_STRING,
      |                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  137 |                                 NO_CARRIED_BY_MONSTER_STRING, allyStatusName, mutationName);
      |                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/brogue/SeedCatalog.c:135:9: note: referencing argument 4 of type 'char[50]'
src/brogue/SeedCatalog.c:135:9: note: referencing argument 5 of type 'char[50]'
src/brogue/SeedCatalog.c:135:9: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:135:9: note: referencing argument 6 of type 'char[50]'
src/brogue/SeedCatalog.c:135:9: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:135:9: note: referencing argument 7 of type 'char[50]'
src/brogue/SeedCatalog.c:135:9: warning: 'printSeedCatalogCsvLine' accessing 10 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:135:9: note: referencing argument 8 of type 'char[10]'
src/brogue/SeedCatalog.c:135:9: warning: 'printSeedCatalogCsvLine' accessing 10 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:135:9: note: referencing argument 9 of type 'char[10]'
src/brogue/SeedCatalog.c:135:9: warning: 'printSeedCatalogCsvLine' accessing 50 bytes in a region of size 1 [-Wstringop-overflow=]
src/brogue/SeedCatalog.c:135:9: note: referencing argument 10 of type 'char[50]'
src/brogue/SeedCatalog.c:135:9: note: referencing argument 11 of type 'char[20]'
src/brogue/SeedCatalog.c:135:9: note: referencing argument 12 of type 'char[100]'
src/brogue/SeedCatalog.c:36:13: note: in a call to function 'printSeedCatalogCsvLine'
   36 | static void printSeedCatalogCsvLine(uint64_t seed, short depth, short quantity, char categoryName[50], char kindName[50],
      |             ^~~~~~~~~~~~~~~~~~~~~~~
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/Time.c -o src/brogue/Time.o
src/brogue/Time.c: In function 'handleHealthAlerts':
src/brogue/Time.c:899:17: warning: 'flashCreatureAlert' accessing 200 bytes in a region of size 79 [-Wstringop-overflow=]
  899 |                 flashCreatureAlert(&player, buf, &badMessageColor, &darkRed);
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/brogue/Time.c:899:17: note: referencing argument 2 of type 'char[200]'
src/brogue/Time.c:865:6: note: in a call to function 'flashCreatureAlert'
  865 | void flashCreatureAlert(creature *monst, char msg[200], const color *foreColor, const color *backColor) {
      |      ^~~~~~~~~~~~~~~~~~
src/brogue/Time.c:916:21: warning: 'flashCreatureAlert' accessing 200 bytes in a region of size 79 [-Wstringop-overflow=]
  916 |                     flashCreatureAlert(&player, buf, &yellow, &darkGreen);
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/brogue/Time.c:916:21: note: referencing argument 2 of type 'char[200]'
src/brogue/Time.c:865:6: note: in a call to function 'flashCreatureAlert'
  865 | void flashCreatureAlert(creature *monst, char msg[200], const color *foreColor, const color *backColor) {
      |      ^~~~~~~~~~~~~~~~~~
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/Utilities.c -o src/brogue/Utilities.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/brogue/Wizard.c -o src/brogue/Wizard.o
In function 'dialogCreateMonster',
    inlined from 'dialogCreateItemOrMonster' at src/brogue/Wizard.c:518:9:
src/brogue/Wizard.c:350:9: warning: 'strncpy' specified bound 100 equals destination size [-Wstringop-truncation]
  350 |         strncpy(buttonText, monsterKinds[i].monsterName, COLS);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In function 'dialogCreateMonsterChooseMutation',
    inlined from 'dialogCreateMonster' at src/brogue/Wizard.c:367:9,
    inlined from 'dialogCreateItemOrMonster' at src/brogue/Wizard.c:518:9:
src/brogue/Wizard.c:287:13: warning: 'strncpy' output may be truncated copying 100 bytes from a string of length 9215 [-Wstringop-truncation]
  287 |             strncpy(buttonText, mutationCatalog[i].title, COLS);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In function 'dialogCreateItem',
    inlined from 'dialogCreateItemOrMonster' at src/brogue/Wizard.c:516:9:
src/brogue/Wizard.c:451:9: warning: 'strncpy' output may be truncated copying 100 bytes from a string of length 142 [-Wstringop-truncation]
  451 |         strncpy(buttonText, itemCategoryNames[i], COLS);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In function 'dialogCreateItemChooseKind',
    inlined from 'dialogCreateItem' at src/brogue/Wizard.c:458:24,
    inlined from 'dialogCreateItemOrMonster' at src/brogue/Wizard.c:516:9:
src/brogue/Wizard.c:187:9: warning: 'strncpy' specified bound 100 equals destination size [-Wstringop-truncation]
  187 |         strncpy(buttonText, kindTable[i].name, COLS);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/variants/GlobalsBrogue.c -o src/variants/GlobalsBrogue.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/variants/GlobalsRapidBrogue.c -o src/variants/GlobalsRapidBrogue.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/platform/main.c -o src/platform/main.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/platform/platformdependent.c -o src/platform/platformdependent.o
In function 'setHighScoresFilename',
    inlined from 'initScores' at src/platform/platformdependent.c:278:5:
src/platform/platformdependent.c:327:5: warning: 'strncat' specified bound 260 equals destination size [-Wstringop-overflow=]
  327 |     strncat(buffer, "HighScores.txt", bufferMaxLength);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/platform/platformdependent.c:326:5: warning: 'strncpy' specified bound 260 equals destination size [-Wstringop-truncation]
  326 |     strncpy(buffer, gameConst->variantName, bufferMaxLength);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In function 'setHighScoresFilename',
    inlined from 'loadScoreBuffer' at src/platform/platformdependent.c:340:5:
src/platform/platformdependent.c:327:5: warning: 'strncat' specified bound 260 equals destination size [-Wstringop-overflow=]
  327 |     strncat(buffer, "HighScores.txt", bufferMaxLength);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/platform/platformdependent.c:326:5: warning: 'strncpy' specified bound 260 equals destination size [-Wstringop-truncation]
  326 |     strncpy(buffer, gameConst->variantName, bufferMaxLength);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In function 'setHighScoresFilename',
    inlined from 'saveScoreBuffer' at src/platform/platformdependent.c:413:5:
src/platform/platformdependent.c:327:5: warning: 'strncat' specified bound 260 equals destination size [-Wstringop-overflow=]
  327 |     strncat(buffer, "HighScores.txt", bufferMaxLength);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/platform/platformdependent.c:326:5: warning: 'strncpy' specified bound 260 equals destination size [-Wstringop-truncation]
  326 |     strncpy(buffer, gameConst->variantName, bufferMaxLength);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/platform/null-platform.c -o src/platform/null-platform.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/platform/sdl2-platform.c -o src/platform/sdl2-platform.o
cc -DDATADIR=. -DBROGUE_EXTRA_VERSION='"-dev"' -DBROGUE_SDL -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2 -c src/platform/tiles.c -o src/platform/tiles.o
/bin/sh: line 1: cmp: command not found
/bin/sh: line 1: cmp: command not found
/bin/sh: line 1: cmp: command not found
cc -Isrc/brogue -Isrc/platform -Isrc/variants -std=c99 -Wall -Wpedantic -Werror=implicit -Wno-parentheses -Wno-unused-result -Wformat -Werror=format-security -Wformat-overflow=0 -IC:/msys64/mingw64/include/SDL2 -Dmain=SDL_main -O2  -o bin/brogue src/brogue/Architect.o src/brogue/Buttons.o src/brogue/Combat.o src/brogue/Dijkstra.o src/brogue/Globals.o src/brogue/GlobalsBase.o src/brogue/Grid.o src/brogue/IO.o src/brogue/Items.o src/brogue/Light.o src/brogue/MainMenu.o src/brogue/Math.o src/brogue/Monsters.o src/brogue/Movement.o src/brogue/PowerTables.o src/brogue/Recordings.o src/brogue/RogueMain.o src/brogue/SeedCatalog.o src/brogue/Time.o src/brogue/Utilities.o src/brogue/Wizard.o src/variants/GlobalsBrogue.o src/variants/GlobalsRapidBrogue.o src/platform/main.o src/platform/platformdependent.o src/platform/null-platform.o src/platform/sdl2-platform.o src/platform/tiles.o -lm -LC:/msys64/mingw64/lib -lmingw32 -mwindows -lSDL2main -lSDL2 -lSDL2_image
```

But I couldn't start the built executable:

```
Sandor@Legion MINGW64 /c/Users/Sandor/Brogue/github/BrogueCE
$ ./bin/brogue.exe
Error: "./assets/tiles.png" not found!
```

Does this indicate that BUILD.md might be outdated?

--- a/BUILD.md
+++ b/BUILD.md
@@ -5,6 +5,7 @@ You need these things at minimum to compile the game:
 
 - a C compiler (Clang or GCC)
 - Make
+- diffutils (cmp)
 
 You will also need additional dependencies based on the build options you select.
 You can see the options and their requirements in `config.mk`. You can configure
@@ -23,13 +24,13 @@ tested by the author and is relatively simple.
 
 ### Setting up
 
-1. Install [MSYS2][1] for x86_64. Refer to [its wiki][2] for essential info
+1. Install [MSYS2][1] for x86_64. Refer to the website for essential info
 on how to use it.
 
 2. (Installing the dependencies) Open the MSYS2 shell and run the following:
 
     ```
-    pacman -S make mingw-w64-x86_64-{gcc,SDL2,SDL2_image}
+    pacman -S make diffutils mingw-w64-x86_64-{gcc,SDL2,SDL2_image}
     ```
 
 ### Building
@@ -100,7 +101,7 @@ These instructions are for the graphical version only.
 1. Install the dependencies; for example, on Debian/Ubuntu, run
 
     ```
-    sudo apt install make gcc libsdl2-2.0-0 libsdl2-dev libsdl2-image-2.0-0 libsdl2-image-dev
+    sudo apt install make gcc diffutils libsdl2-2.0-0 libsdl2-dev libsdl2-image-2.0-0 libsdl2-image-dev
     ```
 
 ### Building
@@ -113,6 +114,5 @@ These instructions are for the graphical version only.
 
 
 [1]: https://www.msys2.org/
-[2]: https://github.com/msys2/msys2/wiki
 [3]: https://www.java.com/en/download/help/path.xml
 [4]: https://brew.sh/

--- a/make/brogue.mk
+++ b/make/brogue.mk
@@ -1,4 +1,4 @@
-bin/brogue$(.exe): $(objects) vars/cflags vars/LDFLAGS vars/libs vars/objects make/brogue.mk
+bin/brogue bin/brogue.exe: $(objects) vars/cflags vars/LDFLAGS vars/libs vars/objects make/brogue.mk
 	$(CC) $(cflags) $(LDFLAGS) -o $@ $(objects) $(libs)
 # on windows, embedding the manifest modifies the executable, preventing debugging
 ifeq ($(SYSTEM),WINDOWS)

